--- a/projects/rocprofiler-systems/cmake/Packages.cmake
+++ b/projects/rocprofiler-systems/cmake/Packages.cmake
@@ -849,20 +849,14 @@ set(TIMEMORY_USE_BFD
 set(TIMEMORY_USE_LIBUNWIND ON CACHE BOOL "Enable libunwind support in timemory")
 set(TIMEMORY_USE_VISIBILITY OFF CACHE BOOL "Enable/disable using visibility decorations")
 
-if(DEFINED TIMEMORY_BUILD_GOTCHA AND NOT TIMEMORY_BUILD_GOTCHA)
-    rocprofiler_systems_message(
-        FATAL_ERROR
-        "Using an external gotcha is not allowed due to known bug that has not been accepted upstream"
-    )
-endif()
-
 # timemory feature build settings
-set(TIMEMORY_BUILD_GOTCHA
-    ON
-    CACHE BOOL
-    "Enable building GOTCHA library from submodule"
-    FORCE
-)
+if(NOT DEFINED TIMEMORY_BUILD_GOTCHA)
+    set(TIMEMORY_BUILD_GOTCHA
+        ON
+        CACHE BOOL
+        "Enable building GOTCHA library from submodule"
+    )
+endif()
 set(TIMEMORY_BUILD_LIBUNWIND
     ${ROCPROFSYS_BUILD_LIBUNWIND}
     CACHE BOOL
@@ -955,11 +949,13 @@ if(_timemory_sanitizer_flag_pos GREATER -1)
     set(CMAKE_CXX_FLAGS "${_timemory_saved_cxx_flags}")
 endif()
 
-install(
-    TARGETS gotcha
-    DESTINATION ${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}
-    COMPONENT gotcha
-)
+if(TIMEMORY_BUILD_GOTCHA AND TARGET gotcha)
+    install(
+        TARGETS gotcha
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}
+        COMPONENT gotcha
+    )
+endif()
 if(ROCPROFSYS_BUILD_LIBUNWIND)
     install(
         DIRECTORY ${PROJECT_BINARY_DIR}/external/timemory/external/libunwind/install/lib/
